frontend/src/scenes/insights/insightLogic.ts

```diff
@@ -668,6 +668,15 @@ export const insightLogic = kea<insightLogicType>({
                 )
             },
         ],
+
+        showTotalCount: [
+            (s) => [s.filters],
+            (filters: Partial<FilterType>): boolean => {
+                const entities = (filters.events || []).concat(filters.actions ?? [])
+                const isOnlyTotals = entities.every((entity) => entity.math === 'total' || !entity.math)
```

Comment:
Let's add a comment here explaining why it's dangerous to sum e.g. averages or unique users.